mappings/net/minecraft/item/DefaultFoodSettings.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS aya net/minecraft/item/FoodItemSettings
+CLASS aya net/minecraft/item/DefaultFoodSettings
```

Comment:
Should we name `Items` `DefaultItems` as well ?

Reply:
Good point. Maybe we should re-evaluate the overall class name. Do you think `FoodMetadata` could work? Then again.. the plural `FoodMetadatas` feels a little odd.

Reply:
Good question. `FoodMetadata` seems alright, alternatives are `FoodInformation` and `FoodComponent`.

Reply:
I like `FoodComponent`, it better describes its use in Mojang's step towards a component system for behavioural attributes.

Reply:
On the other hand, they have a hardcoded getter in Item... Not sure if `item.getFoodComponent()` is very nice, and `item.getFood()` would be too vague.

Reply:
That's what I was thinking. The tweet introducing this change did mention ECS.

Reply:
I don't have a problem with `item.getFoodComponent()` but there may more fluent names

Reply:
Resolved by 794bc6e. Feedback is welcome on the getter name in `Item`.